connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -874,6 +874,14 @@ class SparkConnectPlanner(val session: SparkSession) {
             s"Predicates are not supported for ${rel.getDataSource.getFormat} data sources.")
         }
 
+      case proto.Read.ReadTypeCase.NAMED_TABLE if rel.getIsStreaming =>
+        val multipartIdentifier =
+          CatalystSqlParser.parseMultipartIdentifier(rel.getNamedTable.getUnparsedIdentifier)
+        UnresolvedRelation(
+          multipartIdentifier,
+          new CaseInsensitiveStringMap(rel.getNamedTable.getOptionsMap),
+          isStreaming = true)
```

Comment:
hmm shouldn't this just be `isStreaming = rel.getIsStreaming`?

Reply:
Ah right that makes sense. Thank you!